### ISSUE TITLE
Public vs public access modifier for constructor

### ISSUE THREAD

**Issue by ArabCoders** [kalessil/phpinspectionsea#570]
Hi,

It seems the the access modifier inspection does not accept capital latter example

````php
class ExampleA {
    Public function __construct( array $opts = [] ) { }
}
class ExampleB {
    public function __construct( array $opts = [] ) { }
}
````
ExampleA: will show warning "ExampleA "__construct" should be declared with access modifier"

**Comment by kalessil** [comment]
THank you for reporting @ArabCoders, fixed.
